def raise_exception_msg(msg):
    """Raises a NameError exception with the given message."""
    raise NameError(msg)